opennms-webapp/src/main/java/org/opennms/web/outage/DaoWebOutageRepository.java

```diff
@@ -195,6 +195,7 @@ private Outage mapOnmsOutageToOutage(OnmsOutage onmsOutage) {
             outage.serviceName = onmsOutage.getMonitoredService() != null ? onmsOutage.getMonitoredService().getServiceName() : "";
             outage.suppressedBy = onmsOutage.getSuppressedBy();
             outage.suppressTime = onmsOutage.getSuppressTime();
+            outage.perspectiveLocation = "TODO Patrick";
```

Comment:
@christianpape I'm not sure from where do we get the perspective location?

Reply:
The PR https://github.com/OpenNMS/opennms/pull/3073 isn't merged to features/rpwm yet. We are still waiting for the final approval of @indigo423 . You can of course base your branch off jira/NMS-12792.

Reply:
fixed